### DISCUSSION TITLE
Ресет CustomSelect-а

### DISCUSSION THREAD

**Discussion by revidovich** [VKCOM/VKUI#3820] [Q&A] [answered]
<CustomSelect
              placeholder="Без сортировки"
              options={OPTIONS_SORT}
              value={sorting || ''}
              onChange={(e) => setSorting(e.target.value)}
            />
            
Чтобы сбросить состояние, пишу setSorting('') -- и это мне ничего не даёт, состояние перемещается на первый элемент массива. Приходится первому элементу присваивать нулевое значение, но я не считаю что это нормально. я бы хотела, чтобы при пустой строке (или undefined), чтобы в этом случае селект оказывался пустым и был виден плейсхолдер -- вот такое поведение ожидается.

Пожалуйста, подскажите, собираетесь ли вы обновлять компонент CustomSelect, а может быть я не до конца понимаю кейс?

**Comment by inomdzhon** [comment, marked as answer]
Привет @revidovich 

О проблеме знаем https://github.com/VKCOM/VKUI/issues/3542 😞 

Скорей всего будет кнопка с иконкой в виде крестика. PR есть, приклинкован к issue выше.
